evm_loader/cli/Cargo.toml

```diff
@@ -1,22 +1,27 @@
 [package]
 name = "neon-cli"
-version = "0.5.4"
+version = "0.6.0-rc3"
 authors = ["Cyber-Core Maintainers <maintainers@cyber-core.dev>"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
+chrono = "0.4"
+thiserror = "1.0"
+num-derive = "0.3"
+num-traits = "0.2"
+num_enum = "0.5.1"
```

Comment:
I'm not sure if these crates are currently used by the code. We should check it.

Reply:
It's somehow prevent to create a release?

Reply:
No, it doesn't prevent us from creating the release.
It's just a comment, not requirement.

Reply:
Will you check it in develop branch?

Reply:
I've just checked. These crates are unnecessary. We should remove them next release.

Reply:
OK. Please create an issue for that.

Reply:
Reported.
https://github.com/neonlabsorg/neon-evm/issues/512